pkg/api/scylla/v1alpha1/types_monitoring.go

```diff
@@ -162,6 +162,16 @@ type ScyllaDBMonitoringSpec struct {
 
 	// components hold additional config for the monitoring components in use.
 	Components *Components `json:"components"`
+
+	// type determines what platform service model the monitoring is deployed for. Defaults to Platform. Valid options
+	// are Platform and SaaS.
+	// "Platform" means the monitoring is deployed in the ScyllaDB cluster owner's platform. A full monitoring setup
+	// designed for cluster administrators is deployed.
+	// "SaaS" means the monitoring is deployed for a SaaS offering. A simplified monitoring setup is deployed.
+	// +kubebuilder:default:="Platform"
+	// +kubebuilder:validation:Enum="";"SaaS";"Platform"
```

Comment:
why ""?

Reply:
I thought it was necessary to pass validation with implicit value. Removed, thanks.